pkg/extender/extender.go

```diff
@@ -847,3 +913,168 @@ func (e *Extender) processCSIExtPodPrioritizeRequest(
 		storklog.PodLog(pod).Errorf("Failed to encode response: %v", err)
 	}
 }
+
+func (e *Extender) processVirtLauncherPodPrioritizeRequest(
+	encoder *json.Encoder,
+	args schedulerapi.ExtenderArgs) error {
+	pod := args.Pod
+
+	vmiName, vmiUID := e.getVMIInfo(pod)
+	if vmiName == "" {
+		return fmt.Errorf("unable to find vmiName for pod: %v", pod.Name)
+	}
+	podBeingLiveMigrated, err := e.getLiveMigrationInfo(pod, vmiUID)
+	if err != nil {
+		return fmt.Errorf("unable to find liveMigrationInfo: %w", err)
+	}
+
+	if podBeingLiveMigrated != nil {
+		storklog.PodLog(pod).Debugf("Live VM migration in progress: (vmiName: %v, podBeingLiveMigrated: %v)", vmiName, podBeingLiveMigrated.Name)
+	} else {
+		storklog.PodLog(pod).Debugf("No ongoing live VM migration in progress. vmiName: %v", vmiName)
+	}
+
+	vmi, err := kvd.Instance().GetVirtualMachineInstance(context.TODO(), pod.Namespace, vmiName)
+	if err != nil {
+		return fmt.Errorf("unable to find VMI Info: %w", err)
+	}
+	storklog.PodLog(pod).Debugf("VMI is using PVC %v: ", vmi.RootDiskPVC)
+
+	pvId, err := e.Driver.GetVolumeIDFromPVC(vmi.RootDiskPVC, pod.Namespace)
+	if err != nil {
+		return fmt.Errorf("unable to inspect PVC: %v err: %w", vmi.RootDiskPVC, err)
+	}
+	storklog.PodLog(pod).Debugf("Volume id for scoring: %v", pvId)
+
+	volInfo, err := e.Driver.InspectVolume(pvId)
```

Comment:
FYI: You can query here with PV Name itself.

Reply:
Will take care of this in the next revision.